.gitignore

```diff
@@ -1,11 +1,13 @@
 node_modules
 build
+dist
 
 .DS_Store
 .env
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+*.tsbuildinfo
```

Comment:
Btw, what generates this?

Reply:
Incremental TypeScript builds. It's not actually super relevant to this setup because ts-node-dev is used.